gcp_variant_transforms/testing/integration/vcf_to_bq_tests/huge_tests/test_1000_genomes.json

```diff
@@ -5,22 +5,314 @@
     "input_pattern": "gs://genomics-public-data/1000-genomes/vcf/*.vcf",
     "infer_headers": "True",
     "allow_incompatible_records": "True",
-    "sharding_config_path": "gcp_variant_transforms/testing/data/sharding_configs/integration_homo_sapiens.yaml",
```

Comment:
Please remove `integration_homo_sapiens.yaml` from the repo.

Reply:
Done.